left_column.md

```diff
@@ -77,7 +77,9 @@ The focus of this workshop is on:
   Group work: Lesson design
   (Radovan)
 
-- Preparation for day 2: to be added
+- Preparation for day 2:
+  Browse through [HPC Carpentry material](https://github.com/hpc-carpentry),
+  select and prepare a 5-minute segment which you will teach next day as part of a group exercise.
```

Comment:
```suggestion
  select and prepare a 5-minute segment which you will teach next day as part of a group exercise.
  The section you pick should require screen sharing and be of some follow-along task (preferably
  using a shell).
```
Last time some people picked things that didn't involve interactive type-along, which is not in itself bad, but also not what our main goal with this course is.  Exercise description it the material should also be updated to say this.